Description: Disable jshint
Index: node-grunt-contrib-nodeunit/Gruntfile.js
===================================================================
--- node-grunt-contrib-nodeunit.orig/Gruntfile.js
+++ node-grunt-contrib-nodeunit/Gruntfile.js
@@ -39,7 +39,7 @@ module.exports = function(grunt) {
   grunt.loadTasks('tasks');
 
   // These plugins provide necessary tasks.
-  grunt.loadNpmTasks('grunt-contrib-jshint');
+ // grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-clean');
   grunt.loadNpmTasks('grunt-contrib-internal');
 
@@ -115,5 +115,5 @@ module.exports = function(grunt) {
   });
 
   // By default, lint and run all tests.
-  grunt.registerTask('default', ['jshint', 'nodeunit', 'build-contrib']);
+  grunt.registerTask('default', ['nodeunit', 'build-contrib']);
 };
