Description: Use global tasks
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -40,8 +40,8 @@
 
   // These plugins provide necessary tasks.
  // grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-clean');
-  grunt.loadNpmTasks('grunt-contrib-internal');
+  grunt.loadNpmTasks('grunt-contrib-clean','/usr/lib/nodejs');
+  grunt.loadNpmTasks('grunt-contrib-internal','/usr/lib/nodejs');
 
   // Whenever the "test" task is run, run some basic tests.
   grunt.registerTask('test', function(which) {
